crates/tuono/tests/cli_tests.rs

```diff
@@ -0,0 +1,62 @@
+use std::process::Command;
+use std::fs;
+
+use tempfile::tempdir;
+
+const MAIN_RS_CONTENT: &str = "async fn main() {";
+const INDEX_RS_CONTENT: &str = "async fn get_server_side_props(_req: Request) -> Response {";
+
+// Tests the 'tuono build' command by confirming that main.rs is build and that it contains the start of main() function.
+#[test]
+fn test_tuono_build_success() {
+    let temp_dir = tempdir().expect("Failed to create a temporary test directory.");
```

Comment:
This test will always fail because the `tempdir` you created is always empty. You should create a util function like `fn create_temp_tuono_project() -> TempDir {}` that will be used across all the tesst and that setups a valid tuono project.